### DISCUSSION TITLE
How to select GPTK?

### DISCUSSION THREAD

**Discussion by alshdavid** [italomandara/CXPatcher#135] [Q&A]
I have ticked the "Use GPTK" button 

![image](https://github.com/italomandara/CXPatcher/assets/12656294/cc59f8e5-14af-4994-9a3c-9ff6493add8f)

In tutorials it prompts the user to select the location of the GPTK files - however I receive no such prompt.

Is this no longer the case?

**Comment by italomandara** [comment]
It is no longer the case, the documentation has been updated but when you select that release will show the old docs

**Comment by alshdavid** [comment]
thanks!

Wasn't sure how to know it worked - I guess it's working then 😆
